index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
I think you meant to use `exec()` here and below.

---

> Also I would like to display a message, when falling back to npm, any suggestions on how

@SamVerschueren Any suggestion how to achieve this?

Maybe instead, we would create a new dynamic task if the Yarn one fails.

Example:

```bash
 ✔ Prerequisite check
 ✔ Git
 ✔ Cleanup
 - Installing dependencies using Yarn
     Yarn not installed, falling back to npm
 ⠦ Installing dependencies
 ✔ Running tests
 ✔ Bumping version
 ✔ Publishing package
 ✔ Pushing tags
```